### DISCUSSION TITLE
Solidity 101: Fund Me (Unable to deploy to zksync)

### DISCUSSION THREAD

**Discussion by 10kumaranurag01** [Cyfrin/foundry-full-course-cu#2757] [Q&A]
I am facing an issue where am not able to deploy the smart contract on the Zksync sepolia network using remix.

Here is the video for reference:

[Screencast from 2024-10-27 20-03-49.webm](https://github.com/user-attachments/assets/2f347c10-f37e-414c-a8af-b5de7cb225e9)

Can anyone help me with that?

**Comment by EngrPips** [comment]
Hello @10kumaranurag01, Working with `zksync` can be tough at times, and that is why the `zksync` part of the course is optional. If you are having too much trouble with `zksync`, you can progress, and sepolia is sufficient for everything you will do.
